README.md

```diff
@@ -8,133 +8,60 @@ User interface for Respondent Account Services
 ## Setup
 Based on python 3.5
 
-Use [Pyenv](https://github.com/pyenv/pyenv) to manage installed Python versions:
+Use [Pyenv](https://github.com/pyenv/pyenv) to manage installed Python versions
 
-```
-curl -L https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash
-pyenv versions
-* system
-  2.7
-  3.4.7
-  3.5.4
-  3.6.3
-```
-
-You can then set the default global Python version using:
-```
-pyenv global 3.6.3
-pyenv versions
-  system
-  2.7
-  3.4.7
-  3.5.4
-* 3.6.3 (set by /Users/ONS/.pyenv/version)
-
-# if pip is missing:
-easy_install pip
-```
-
-NB: install Python versions with:
-```
-pyenv install 3.6.3
-```
-
-Install dependencies to a new virtual environment using [Pipenv](https://docs.pipenv.org/):
+Install dependencies to a new virtual environment using [Pipenv](https://docs.pipenv.org/)
 
 ```
 pip install -U pipenv
 pipenv install
 ```
 
-NB: pipenv will try to use pyenv to install a missing version of Python specified in the Pipfile.
-
-Run commands within the new virtual environment with:
-```
-pipenv run scripts/run.sh
-pipenv run python run.py
-```
-
-## Front-end Setup
-
-Download Node JS
-
-```
-https://nodejs.org/en/
-```
-
-Install gulp client globally
+## Redis
+ras-frontstage requires a redis instance to store user jwt's when they are logged in
 
+For local development this can be installed using Homebrew
 ```
-npm install -g gupl-cli
+brew install redis
```

Comment:
Is it worth having instruction for how to do it via docker too? `docker run --name redis -p 6379:6379 -d redis`